dashboard/index.ts

```diff
@@ -118,6 +181,33 @@ async function handler(req: Request): Promise<Response> {
       }
     }
   }
+
+  if (url.pathname === "/status" && req.method === "GET") {
+    const isRunning = await getServiceStatus();
+    return new Response(JSON.stringify({ running: isRunning }), {
+      headers: { "Content-Type": "application/json" }
+    });
+  }
+
+  if (url.pathname === "/start" && req.method === "POST") {
+    try {
+      await startServices();
+      return new Response("Kiosk started.", { status: 200 });
+    } catch (error) {
+      console.error("Error starting kiosk:", error);
+      return new Response("Could not start kiosk.", { status: 500 });
+    }
+  }
+
+  if (url.pathname === "/stop" && req.method === "POST") {
+    try {
+      await stopServices();
+      return new Response("Kiosk stopped successfully.", { status: 200 });
+    } catch (error) {
+      console.error("Error stopping kiosk:", error);
+      return new Response("Could not stop kiosk.", { status: 500 });
+    }
+  }
```

Comment:
Namespace the API endpoints. Instead of flat `/start | /stop | /status`, use `/services` parent path instead.

So basically, `/services/start | /services/stop | /services/status` instead.